Running-Mastodon/Development-guide.md

```diff
@@ -27,8 +31,14 @@ To setup the `mastodon_development` database, run:
 
 You can then run Mastodon with:
 
+    ./bin/webpack-dev-server & # New after 1.3.3
```

Comment:
Probably not ideal to detach this process, log output will be really messy. Just recommend using different tabs in the terminal or two terminals. If someone wants to use detaching, they'll know how to anyway.